portal-impl/src/com/liferay/portlet/documentlibrary/service/permission/DLFileEntryPermission.java

```diff
@@ -110,6 +118,9 @@ public static boolean contains(
 			if (hasPermission != null) {
 				return hasPermission.booleanValue();
 			}
+			else if (actionId.equals(ActionKeys.VIEW)) {
```

Comment:
Hey guys,

Why are you forbidding access when the user tries to view the doc, but not in other cases? Are you doing this because of the changes introduced in LPS-10500 (`WorkflowPermissionImpl`, around line 83)? Or is there another reason?